Composer/packages/webchat-client/src/utils/ConversationService.ts

```diff
@@ -0,0 +1,316 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
nit: file name should be camleCase, PascalCase ins only for Components

Reply:
Using react.memo now